### PR TITLE
Ryan/chores

### PR DESCRIPTION
**Pull request by ryancragun**
* Update MOTD to point to the proper setup wizards
* License acceptance is done via the setup wizards so add license files to bypass reconfigure checks
* Update chef-ingredient to use packages.chef.io
* Prune the package and chef cache because our root disk can be small
* Automatically stop and start services during upgrade

### CODE REVIEW

**Comment by ryancragun** [comment]
http://wilson.ci.chef.co/job/chef-marketplace-trigger-ad_hoc/28/downstreambuildview/

File: files/chef-marketplace-cookbooks/chef-marketplace/recipes/_upgrade_analytics.rb [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+bash "opscode-analytics-ctl stop" do
+  code "opscode-analytics-ctl stop"
+  only_if { analytics_configured? }
+end
+
 chef_ingredient "analytics" do
   action :upgrade
-  notifies :run, "bash[opscode-analytics-ctl reconfigure]", :immediately if analytics_configured?
+  notifies :run, "bash[opscode-analytics-ctl reconfigure]", :immediately
+  notifies :run, "ruby_block[remove-analytics-from-cache]", :immediately
```
Comment (adamleff) [inline files/chef-marketplace-cookbooks/chef-marketplace/recipes/_upgrade_analytics.rb]: Is this a technical necessity, or just a "clean-up-so-our-image-is-as-small-as-possible" step?

**Comment by adamleff** [comment]
LGTM. 👍

File: files/chef-marketplace-cookbooks/chef-marketplace/recipes/_upgrade_analytics.rb (hunk 1 above)
Comment (ryancragun) [inline files/chef-marketplace-cookbooks/chef-marketplace/recipes/_upgrade_analytics.rb]: It's pretty much a necessity.  The root disk only has maybe 2GB free and our omnibus packages are YUUUGE

Comment (adamleff) [inline files/chef-marketplace-cookbooks/chef-marketplace/recipes/_upgrade_analytics.rb]: Heh - just wanted to make sure this wasn't an ugly hack around something Chef was doing that we didn't like, vs. disk space consumption / good behavior. Thanks for clarifying. :)

**Pull request merged**